src/l1_fetcher.rs

```diff
@@ -139,7 +139,22 @@ impl L1Fetcher {
                 .to_block(current_l1_block_number + BLOCK_STEP);
 
             // Grab all relevant logs.
-            let logs = self.provider.get_logs(&filter).await?;
+            let mut resp: Option<_> = None;
```

Comment:
```suggestion
            let mut resp = None;
```
I'm pretty sure the whole type can be inferred here